### PR TITLE
Spectra (PyPI)

### PR DESCRIPTION
**Pull request by ewels**
Package for [Spectra](https://github.com/jsvine/spectra) - easy color scales and color conversion for Python.

### CODE REVIEW

**Comment by ewels** [comment]
Missing a dependency, hopefully resolved in PR #2990

**Comment by ewels** [comment]
Everything working as expected now - hopefully ready to go!

File: recipes/spectra/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - colormath
```
Comment (jakirkham) [inline recipes/spectra/meta.yaml]: Is `colormath` needed at build time? If not, could you please drop it from `build`?

File: recipes/spectra/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    Saturating/desaturating colors and Conversion to/from multiple color spaces.
+    Spectra is built on colormath and grapefruit. Spectra is enormously
+    inspired by chroma.js and d3's scales.
+  license: MIT
+  license_family: MIT
+  # PyPI missing license file. Added PR to add it to future sdists.
+  #
+  # xref: https://github.com/jsvine/spectra/pull/6
+  #
+  #license_file: LICENSE.txt
```
Comment (jakirkham) [inline recipes/spectra/meta.yaml]: xref: https://github.com/jsvine/spectra/pull/6

**Comment by jakirkham** [comment]
One comment about build dependencies. Otherwise LGTM.

File: recipes/spectra/meta.yaml (hunk 1 above)
Comment (ewels) [inline recipes/spectra/meta.yaml]: Probably not - removed 👍

**Comment by ewels** [comment]
Thanks @jakirkham! Updated accordingly..

**Comment by ewels** [comment]
:tada:

**Comment by jakirkham** [comment]
Thanks @ewels.

**Comment by ewels** [comment]
Great stuff, thanks! The web of dependencies is finally sorted, now I can go and release my own package 🕺 🎉

**Pull request merged**